charts/datadog-operator/values.yaml

```diff
@@ -15,6 +15,9 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# DataDog API endpoint, example: https://api.datadoghq.eu
+dataDogUrl: #<DATADOG_API_ENDPOINT>
```

Comment:
Could you used the same variable name (dd_url) that in the `datadog/datadog` chart? https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml#L67

Reply:
Done!